snorkel/labeling/preprocess/nlp.py

```diff
@@ -22,4 +43,16 @@ def __init__(
         self._nlp = spacy.load(language, disable=disable)
 
     def run(self, text: str) -> FieldMap:  # type: ignore
+        """Run the SpaCy model on input text.
+
+        Parameters
+        ----------
+        text
+            Text of document to parse
+
+        Returns
+        -------
+        FieldMap
+            Mapping containing parsed document
```

Comment:
"Mapping containing parsed document"  doesn't tell me what's being mapped to what. The key names are spacy attribute names and the values are the values of the tokens for that attribute, right?